plugins/chilli/chilli_sessions_

```diff
@@ -0,0 +1,117 @@
+#!/bin/sh
+# -*- sh -*-
+
+: << =cut
+
+=head1 NAME
+
+chilli_sessions_ - Wildcard-plugin to monitor sessions state on Coova Chilli.
+
```

Comment:
Please add a short description (`=head1 DESCRIPTION`) of "chilli" (radius ...) and the scope of this plugin (number of active, blocked and unknown(?)) users/connections/devices(?) for each session (or is there a better word?).